website/data/plugin-nav-data.json

```
@@ -29,6 +29,32 @@
     "title": "Debugging",
     "path": "debugging"
   },
+  {
+    "title": "Best Practices",
+    "routes": [
+      { "title": "Overview", "path": "best-practices" },
+      {
+        "title": "Handling Sensitive Data",
+        "path": "best-practices/sensitive-state"
+      },
+      {
+        "title": "Interacting with Providers",
+        "path": "best-practices/interacting-with-providers"
+      },
+      {
+        "title": "Naming",
+        "path": "best-practices/naming"
+      },
+      {
+        "title": "Provider Code Language",
+        "path": "best-practices/other-languages"
+      },
```

Now might be a good time to consider reframing this page so it can more broadly talk about:

- What code languages HashiCorp provides first-class support with provider development (Go only, today)
- What libraries are supported within those code languages

Updating the navigation and source file name to something like this might help convey what I mean:

```suggestion
      {
        "title": "Provider Code",
        "path": "best-practices/provider-code"
      },
```

Along with structuring the page content around this:

- HashiCorp Supported Languages and Libraries
  - Go
    - SDK
    - Framework (preferred)
- Other Languages

---

Updated the page + nav. Will work on restructuring the content next

---

@bflad - take another look at the provider-code page. I updated it with more detail about the different SDKs / support: https://github.com/hashicorp/terraform-docs-common/pull/295/files#diff-f8bbed7ec637c8381fc3af17ae6854b6da859b37656d065762df426cc2d7a952